README.md

```diff
@@ -195,24 +200,144 @@ Key       | Type    | Default     | Description
 
 Key       | Type     | Default     | Description
 :--       | :--      | :--         | :--
-`in`      | Array    | `undefined` | An array of strings to valid paths for CSS files. All files will remain untouched. A new atomized string is produced for `out`/`result`.
+`in`      | Array    | `undefined` | An array of strings to valid paths for CSS files. All files will remain untouched. A new atomized string is produced for `out`/hooks.
+`data`    | String   | `undefined` | A string of CSS to be atomized. Files are provived via `in` are concatenated with `data` at the end, then atomized and sent to `out`/hooks.
 `out`     | String   | `undefined` | A string file path output. If file exists it will be overwritten with the atomized styles from `in` and/or `data`
```

Comment:
```suggestion
`out`     | String   | `undefined` | A string file path output. If file exists it will be overwritten with the atomized styles from `in` and/or `data`
`hooks`   | Object   | `{}`        | Lifecycle callback hooks (documented in next section)
```